app/_landing_pages/kubernetes-ingress-controller.yaml

```diff
@@ -131,10 +131,76 @@ rows:
+                  * [Feature gates](/kubernetes-ingress-controller/reference/feature-gates/): If something isn't working as expected, ensure that you have the correct feature gates activated.
                   * [Create a GitHub issue](https://github.com/Kong/kubernetes-ingress-controller/issues/new/choose): Report a bug or make a feature request for {{ site.kic_product_name }}.
                   * [Join the Kong Slack community](https://konghq.com/community#64fe8580b1a2f3c3804230f1): Get help from the team
 
+  - header:
+      type: h2
+      text: KIC feature support and deployment models
+    columns:
+      - blocks:
+          - type: text
+            config: |
+              {{ site.kic_product_name }} integrates  {{ site.base_gateway }} with Kubernetes by translating Kubernetes resources into {{ site.base_gateway }} configuration.
+              Feature support differs depending on the operational model:
+              - **Standalone (KIC only)**: You provision and manage  {{ site.base_gateway }} yourself, and {{ site.kic_product_name }} applies configuration using Kubernetes resources.
+              - **With Kong Gateway Operator (KO/KGO)**: The Operator provisions and manages {{ site.base_gateway }} deployments declaratively, while {{ site.kic_product_name }} handles configuration. 
+              
+              To compare feature support in each model, use the following table:
+              
+              {% table %}
+              columns:
+                - title: Feature
+                  key: feature
+                - title: KIC-only Model
+                  key: kic
+                - title: KO/KGO Hybrid Model
+                  key: hybrid
+              rows:
+                - feature: "**Configuration**"
+                  kic: "Configure all {{ site.base_gateway }} entities with CRDs directly. For example, Services, Routes, Consumers, Plugins, Certificates, CA Certificates, Vaults."
+                  hybrid: "Same configuration scope, however the Operator automatically provisions and reconciles the gateways."
+                - feature: "**{{ site.base_gateway }} API**" 
+                  kic: "Supports `Ingress`, `HTTPRoute`, `TCPRoute`, and `UDPRoute`. From KIC 3.5 onward, compliant with Gateway API v1.3."
+                  hybrid: "Supports `GatewayClass`, `Gateway`, and `GRPCRoute`, in addition to `HTTPRoute`, `TCPRoute`, and `UDPRoute`."
+                - feature: "**Feature Gates**"  
+                  kic: |
+                    Activate gates like  `FillIDs`, `RewriteURIs`, and `CombinedServicesFromDifferentHTTPRoutes` with `CONTROLLER_FEATURE_GATES`.
+                    - `FillIDs` can cause UUID collisions across workspaces with identical Consumers.
+                    - Deactivate `CombinedRoutes` if you require per-route plugin configuration.
+                  hybrid: "Identical feature gate support and constraints. No change between models."
```

Comment:
Identical feature gate support and constraints declared through CRDs. No change between models